### DISCUSSION TITLE
Visual Studio as default editor

### DISCUSSION THREAD

**Discussion by deasn789** [community/community#23238] [New to GitHub]
<p>Hello,</p>
<p>I am new to Github. I just downloaded Github desktop. I want to change my default editor to Visual Studio. When I follow the directions and go to options and then click on the drop-down to change the default editor only notepad++ is showing as an option for my default editor. I have Visual studio 2019 community version on my computer and have been using it for school and I want to set it as my default editor for Github desktop. I am learning C++ currently. </p>

**Comment by yamiacat** [comment, marked as answer]
<p>Hi deasn789! I’m afraid only some editors can be set as the default option in Desktop. VS Code is supported, but Visual Studio itself is not, sorry!</p>

**Comment by dcube9** [comment]
<p>Hi yamiacat,<br>
ok, now VS2019 is not supported, but in the future ?<br>
It’s possibile add it as a new feature for next release ?</p>
<p>Best regards<br>
Paolo</p>

**Comment by yamiacat** [comment]
<p>Hi <a class="mention" href="/dcube9">@dcube9</a></p>
<p>It’s possible! We’re always working to improve GitHub and the GitHub Support Community, and we consider every suggestion we receive.</p>
<p>Would you mind <a href="https://support.github.com/contact/feedback">submitting this through our official product feedback form</a> so that our product team can track your request?</p>
<p>Thanks!</p>

**Comment by dcube9** [comment]
<p>Hi <a class="mention" href="/yamiacat">@yamiacat</a>,<br>
thank you for reply.<br>
I just submit it.<br>
Best regards<br>
Paolo</p>

**Comment by kalicialopez** [comment]
Checking in in 2024, has this still not been enabled? I've been searching and searching for the right commands to set visual studio as my default IDE and because of Visual Studio Code and Visual Studio having similar names, it has been hard to find confirmation about the state of this, until this thread. I can only assume this suggestion has not been action yet, with the new Visual Studio 2022...
